vpdq/cpp/hashing/filehasher.cpp

```diff
@@ -63,7 +63,12 @@ bool hashVideoFile(
   int numRGBTriples = height * width;
   int fno = 0;
   unique_ptr<uint8_t[]> rawFrameBuffer(new uint8_t[numRGBTriples * 3]);
-
+  int frameMod = secondsPerHash * framesPerSec;
```

Comment:
nit: Add comment that you are doing an intentional  floor operation. I think it is kind of easy to miss given the `double` type declaration  for these two vars is a a fair bit higher up in the file.